### DISCUSSION TITLE
Is it /roʊz paɪn/ or /rɔsé paɪn/?

### DISCUSSION THREAD

**Discussion by kilofox** [rose-pine/rose-pine-theme#112] [General]
Just curious about the pronunciation. Is it "rose" as in the English word rose, or is it "rosé", as in the correct pronunciation of "rosé gold"?

**Comment by mvllow** [comment]
My intention was a bottle of Rosé wine and the smell of pine trees. I hear and see Rose (no accent) Pine quite often though which makes sense—our repositories and filenames are url friendly, usually being "rose-pine" and such.

My partner and I will sometimes jokingly pronounce it in Italian sounding more like: Rosé Pine (peen/aye).

Good question :)
